src/EditorFeatures/CSharpTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.cs

```diff
@@ -320,6 +320,31 @@ class Class1 { }
             await TestMoveTypeToNewFileAsync(code, codeAfterMove, expectedDocumentName, destinationDocumentText);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveType)]
+        public async Task MoveTypeWithWithFileScopedNamespace()
```

Comment:
Can you add a tests for moving Class1 and Class2 with this setup? Assuming this is valid with filescoped namespaces

```csharp
namespace N1;

class Class1 
{
    Nested.Class2 C2 { get; }
}

namespace Nested
{
    class Class2 { }
}
```

Reply:
that's not valid.  it's illegal to mix/match a file-scoped namespace with any other sort of namespace (including another file scoped namespace).